frontend/src/routes/(authenticated)/admin/ProjectTable.svelte

```diff
@@ -53,6 +66,18 @@ async function softDeleteProject(project: Project): Promise<void> {
         notifyWarning($t('delete_project_modal.success', {name: project.name, code: project.code}));
     }
 }
+
+  const migrationStatusTable = {
```

Comment:
`Table` looks like an oversight maybe?
```suggestion
  const migrationStatusToIcon = {
```

Reply:
Sure, why not.

Reply:
Done in commit https://github.com/sillsdev/languageforge-lexbox/commit/ae57e274b20a511839df463862db09b76406bcc0.